src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (averageAge, people) => {
```

Comment:
bad name for parameter, your function has name calculateAverageAge but it already accept average age as parameter. It is confusing.